handler/enqueue_annotation_suite_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
This file applies to all tests in this package, so rename to `handler_suite_test.go`

If/when we add other handlers, it should not be named specific to the enqueue annotation handler.